src/test/java/schools/SchoolTest.java

```diff
@@ -27,76 +25,110 @@ public void clearDataAfterTest(){
         school.getSchoolClasses().clear();
     }
 
+    /**
+     * uwaga ogolna - za mala probla danych testowych i ich przypadkow. Czy na pewno dobrze zadzialaja metody liczace ucznikow jak bedzie ich wiecej niz 1 w klasie
+     * i wiecej klas? Na razie nie ma takiej pewnosci.
+     * A co te klasy maja zwrocic jak nie ma zadnej klasy?
+     * Czy w ogole mozna utworzyc szkole bez klasy?
+     * Czy mozna dodac klase bez zadnego ucznia?
+     *
+     * Jakiekolwiek sa odpowiedzi na ww pytania powinny byc dla nich testy.
+     */
     @Test
-    public void shouldReturnFalseIfClassExist(){
+    // dobrze by nazwy reprezentowaly efekt biznesowy a nie techniczny. Oczywiscie czasem sie nie da ale akurat tutaj
+    // jest mozliwosc opisania efektu biznesowego
+    public void shouldNotLetAddTheSameClassMoreThanOne(){
         // given
         school.addClass(clazzA);
+        // jak bedzie taka operacja robiona na klasach to musimy je zwrocic
```

Comment:
klasę dodałam aby móc później ją usunąć , nie wiem jak inaczej bez tej operacji mogłabym sprawdzić metodę add. Chyba że źle rozumiem komentarz

Reply:
eh, faktycznie to mialo byc troche nizej. Chodzilo mi o school.getSchoolClasses() jakie jest w kolejny tescie.